docs/release-notes.md

```diff
@@ -2,6 +2,37 @@
 title: Release Notes
 ---
 
+## v2.1.0
+
+### New Features
+
+- Increase scalability for [multi-gateway scenarios](./user-guide/scalability.md).
+  This eliminates several disadvantages in a [multi-LNS deployment scenario](./user-guide/deployment-scenarios.md).
+  See [ADR - 010. LNS sticky affinity over multiple sessions](./adr/010_lns_affinity.md).
+- [Standalone discovery service](./user-guide/lns-discovery.md) for dynamic LNS discovery.
+  See [ADR - 009. LoRaWAN Network Server (LNS) discovery](./adr/009_discovery.md).
+- LoRaBasicsStationModule v2.1.0 includes:
+  - Binaries for Basics Station v2.0.6:
+  - Support for SX1302 ([configurable](./user-guide/station-module-configuration.md) via "CORECELL" parameter)
+  - Adjustable log level ([configurable](./user-guide/station-module-configuration.md) via "LOG_LEVEL" parameter)
+
+### Breaking Changes
+
+- [#1576](https://github.com/Azure/iotedge-lorawan-starterkit/issues/1576): The default deduplication strategy is now "Drop" instead of "None". More information found in the [decision record](./adr/007_message_deduplication.md).
```

Comment:
Let's add the new API Version 2022_03_04, which is only compatible with v2.1.0 LNS (Direct method for closing open connections)

Reply:
Good idea, done in c72011a7753cc55d596bbcae2ff0cbd39a3acca7